packages/app/src/cli/services/dev/processes/theme-app-extension-next.ts

```diff
@@ -31,8 +31,6 @@ export interface PreviewThemeAppExtensionsProcess extends BaseProcess<ThemeAppEx
 export async function setupPreviewThemeAppExtensionsProcess(
   options: HostThemeSetupOptions,
 ): Promise<PreviewThemeAppExtensionsProcess | undefined> {
-  outputInfo('This feature is currently in development and is not ready for use or testing yet.')
```

Comment:
Are we removing this intentionally? I believe there's more work to be done on the TAE side, but I may be mistaken

Reply:
It was intentional, but it's indeed a better idea to remove this line [here](https://github.com/Shopify/cli/pull/4375) :)